de-lib/src/main/java/org/iplantc/de/desktop/client/views/DesktopViewImpl.java

```diff
@@ -53,22 +53,24 @@
     @UiField IconButton analysisWinBtn;
     @UiField IconButton appsWinBtn;
     @UiField IconButton dataWinBtn;
-    @UiField IconButton feedbackBtn;
-    @UiField IconButton forumsBtn;
+    @UiField DesktopIconButton helpBtn;
     @UiField IconButton notificationsBtn;
     @UiField TaskBar taskBar;
     @UiField DesktopIconButton userSettingsBtn;
-    @UiField IPlantAnchor preferencesBtn;
-    @UiField IPlantAnchor collaboratorsBtn;
-    @UiField IPlantAnchor systemMsgsBtn;
-    @UiField IPlantAnchor documentationBtn;
+    @UiField IPlantAnchor preferencesLink;
+    @UiField IPlantAnchor collaboratorsLink;
+    @UiField IPlantAnchor systemMsgsLink;
+    @UiField IPlantAnchor documentationLink;
     @UiField IPlantAnchor introBtn;
-    @UiField IPlantAnchor contactSupportBtn;
+ //   @UiField IPlantAnchor contactSupportBtn;
```

Comment:
Ahh I just forgot to get rid of them. I started with commenting out stuff that wasn't needed.